MakeHist/mumu.C

```diff
@@ -0,0 +1,337 @@
+
+   TH1F *SBM_hist = new TH1F("SBM_hist","max(SBM)",100,0,300); SBM_hist->Sumw2();
+   TH1F *METvsMmm_hist = new TH1F("METvsMmm_hist","E^{miss}_{T}/M(#mu^{+}#mu^{-})",100,0,1); METvsMmm_hist->Sumw2();
+   TH1F *HTLT_hist = new TH1F("HTLT_hist","HT-LT",100,-500,500); HTLT_hist->Sumw2();
+
+   TH1F *Mass_hist = new TH1F("Mass_hist","Lepton mass(M_{#mu})",100,0,500); Mass_hist->Sumw2();
+
+   Long64_t nentries = fChain->GetEntriesFast();
+
+   //totalN, Lumi -> Refer to see header file (.h)
+   float cs = 76.7 * 1000;
+   float weight = Lumi * cs/TT_totalN;
+
+   Long64_t nbytes = 0, nb = 0;
+   //Loop over all events
+   for (Long64_t jentry=0; jentry<nentries;jentry++) {
+       Long64_t ientry = LoadTree(jentry);
+       if (ientry < 0) break;
+       nb = fChain->GetEntry(jentry);   nbytes += nb;
+       // if (Cut(ientry) < 0) continue;
+
+       run_hist->Fill(run);
+
+       if(lep_pt[0] > 53 &&lep_pt[1] > 53){ //lep pT > 53GeV
+           if(dil_dR > 0.1){ // dR(lep,lep) > 0.1
+               if(fabs(lep_dz[0])<0.5 && fabs(lep_dz[1])<0.5){
+
+                   //Opposite sign dilepton
+                   if(lep_id[0] * lep_id[1] < 0){
+
+                       //if(dil_chosen != 0) continue;
+                       //particle id
+                       int id1 = lep_id[0];
+                       int id2 = lep_id[1];
+
+                       id_hist->Fill(id1);
+                       id_hist->Fill(id2);
+
+                       const double muon_mass = 0.1056583715;
+
+                       //Save leptons information
+                       vector<TLorentzVector> leptons;
+                       vector<Lepton> leptons_;
+                       for(unsigned lep_N=0; lep_N<2; ++lep_N){
+                           TLorentzVector lepton;
+                           TLorentzVector lepton_;
+                           lepton.SetPtEtaPhiM(lep_pt[lep_N],lep_eta[lep_N],lep_phi[lep_N],muon_mass);
+                           lepton_.SetPtEtaPhiM(lep_pt[lep_N],lep_eta[lep_N],lep_phi[lep_N],muon_mass);
+                           leptons.push_back(lepton);
+                           Lepton dummy = {lepton_, lep_pfIso[lep_N]};
+                           leptons_.push_back(dummy);
+                       }  // example(1): float FirstLeptonMass = leptons[0].M();
+                       ///// example(2): float FirstLeptonIso = leptons_[0].Iso;
+
+                       float Dil_dPhi[1] = {-999};
+                       float Dil_dR[1] = {-999};
+                       float Dil_mass[1] = {-999};
+
+                       float MET[1] = {-999};
+
+                       int Njet = 0;
+                       int Nbjet = 0;
+
+                       vector<TLorentzVector> Jets;
+                       vector<TLorentzVector> bJets;
+
+                       for(int NJ=0; NJ<nJets; NJ++){
+                           if(jet_pt[NJ] > 30){ //jet pT > 20GeV
```

Comment:
We promised to go down to 20 GeV for b-tagged jets, so this will have to be changed to 20

Reply:
I will change 30 GeV cut to 20 GeV